.storybook/blocks/Status/Status.module.css

```diff
@@ -9,12 +9,7 @@
   background-color: var(--color-tarmac-grey-50);
 }
 
-.in_development {
-  color: var(--color-sunbeam-blue-900);
-  background-color: var(--color-sunbeam-blue-100);
-}
-
-.in_review {
+.ready {
```

Comment:
Could you pls change the `ready` color to the green `--color-success-200` and the `Recommendation ` color to `--color-tertiary-200` or `--color-tertiary-300`just check what will be a better fit for it.